### PR TITLE
[TR] Bunch of fixes related to PEF

### PR DESCRIPTION
**Pull request by grena**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | N

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form.js [hunk 1]
```diff
@@ -261,6 +261,24 @@ define(
...
+             * Listen to given mediator events to trigger them locally (in the local root).
+             * This way, extensions attached to this form don't have to listen "globally" on the mediator.
+             *
+             * @param {Array} mediator events to forward:
+             *                [ {'mediator:event:name': 'this:event:name'}, {...} ]
+             */
+            forwardMediatorEvents: function (events) {
+                _.each(events, function (eventInfo) {
+                    var mediatorEvent = _.keys(eventInfo)[0];
+                    var localEvent = _.values(eventInfo)[0];
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form.js]: align ? =)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes/copy.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ define(
                 this.locale = UserContext.get('catalogLocale');
                 this.scope  = UserContext.get('catalogScope');
 
+                //this.listenTo(this.getRoot(), 'pim_enrich:form:field:extension:add', this.addFieldExtension);
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes/copy.js]: Why not replacing this one ?

Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/attributes/copy.js]: Nice catch ! Thanks

**Comment by phaseinducer** [comment]
GTM after fix, nice PR :+1:

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form.js [hunk 3]
```diff
@@ -261,6 +261,24 @@ define(
...
+            /**
+             * Listen to given mediator events to trigger them locally (in the local root).
+             * This way, extensions attached to this form don't have to listen "globally" on the mediator.
+             *
+             * @param {Array} mediator events to forward:
+             *                [ {'mediator:event:name': 'this:event:name'}, {...} ]
+             */
+            forwardMediatorEvents: function (events) {
+                _.each(events, function (eventInfo) {
+                    var mediatorEvent = _.keys(eventInfo)[0];
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form.js]: _.first ?

**Comment by sumbobyboys** [comment]
Less mediator, such forward, much GTM

**Pull request merged**